frontend/src/modules/alarms/AlarmsPage.tsx

```diff
@@ -292,41 +294,71 @@ export const AlarmsPage = (): JSX.Element => {
     setPage(value);
   };
 
-  const alarmLimitsRequestStandby = useSelector(getAlarmLimitsRequestStandby);
+  const alarmLimitsRequestDraft = useSelector(getAlarmLimitsRequestDraft);
+  const alarmLimitsCurrent = useSelector(getAlarmLimitsCurrent);
   const dispatch = useDispatch();
   const currentMode = useSelector(getParametersRequestMode);
   const ventilating = useSelector(getParametersIsVentilating);
-  const setAlarmLimits = (data: Partial<AlarmLimitsRequest>) => {
+  const alarmLimitsUnsaved = useSelector(getAlarmLimitsUnsavedChanges);
+  const [alarmLimits] = useState((alarmLimitsCurrent as unknown) as Record<string, Range>);
+  const alarmLimitsDraft = (alarmLimitsRequestDraft as unknown) as Record<string, Range>;
+  const setAlarmLimitsRequestDraft = (data: Partial<AlarmLimitsRequest>) => {
     dispatch(commitStandbyRequest<AlarmLimitsRequest>(MessageType.AlarmLimitsRequest, data));
   };
   const applyChanges = () => {
-    if (alarmLimitsRequestStandby === null) {
+    if (alarmLimitsRequestDraft === null) {
       return;
     }
 
     dispatch(
-      commitRequest<AlarmLimitsRequest>(MessageType.AlarmLimitsRequest, alarmLimitsRequestStandby),
+      commitRequest<AlarmLimitsRequest>(MessageType.AlarmLimitsRequest, alarmLimitsRequestDraft),
     );
   };
   const alarmConfig = alarmConfiguration(currentMode);
   const [open, setOpen] = useState(false);
```

Comment:
Is this actually supposed to mean `confirmOpen` and `setConfirmOpen`?

Reply:
yes. done